AllJoyn/Platform/TestApps/UwpTestApp/MainPage.xaml.cs

```diff
@@ -105,5 +141,85 @@ private void InitButton_Click(object sender, RoutedEventArgs e)
                 }
             }
         }
+
+        private void OpenT2TBridgeArchitestrationService_ServiceClosed(AppServiceConnection sender, AppServiceClosedEventArgs args)
+        {
+            msg.Text = "App Service closed";
+            openT2TBridgeArchitestrationService = null;
+        }
+
+        private void OpenT2TBridgeArchitestrationService_RequestReceived(AppServiceConnection sender, AppServiceRequestReceivedEventArgs args)
+        {
+            throw new NotImplementedException();
+        }
+
+        [DataContract]
+        internal class DeviceProps
+        {
+            public static DeviceProps CreateInstance(string id, string accessToken)
+            {
+                return new DeviceProps(id, accessToken);
+            }
+            private DeviceProps(string identifier, string accessToken)
+            {
+                id = identifier;
+                access_token = accessToken;
+            }
+
+            [DataMember]
+            internal string id;
+
+            [DataMember]
+            internal string access_token;
+        }
+        // TODO : Currently Vendor/Model are not sent to app service, which also do not look for them. 
+        private async Task<ValueSet> GetBridgeAppServiceValueSet(string name, string props, string jsFile, string schemaFile)
+        {
+            DataContractJsonSerializer js = new DataContractJsonSerializer(typeof(DeviceProps));
+            MemoryStream ms = new MemoryStream(System.Text.ASCIIEncoding.ASCII.GetBytes(props));
```

Comment:
sure. I shall test that out.

---
In reply to: [75189001](https://github.com/tjaffri/msiot-samples/pull/9#discussion_r75189001) [](ancestors = 75189001)